pkg/install/generate.go

```diff
@@ -333,19 +342,17 @@ func InstallerPodSpec(
 			VolumeMounts: volumeMounts,
 		})
 	}
+
 	containers := []corev1.Container{
 		{
-			Name:            "hive",
-			Image:           images.GetHiveImage(),
-			ImagePullPolicy: images.GetHiveClusterProvisionImagePullPolicy(),
+			Name:            "installer",
+			Image:           installerImage,
+			ImagePullPolicy: corev1.PullIfNotPresent,
 			Env:             append(env, cd.Spec.Provisioning.InstallerEnv...),
-			Command:         []string{"/usr/bin/hiveutil"},
-			Args: []string{
-				"install-manager",
-				"--work-dir", "/output",
-				"--log-level", "debug",
-				cd.Namespace, provisionName,
-			},
+			Command:         []string{"/bin/sh", "-c"},
+			// Large file copy here has shown to cause problems in clusters under load, safer to copy then rename to the file the install manager is waiting for
+			// so it doesn't try to run a partially copied binary.
+			Args:         []string{fmt.Sprintf("cp -v /bin/openshift-install /output/openshift-install && major_version=$(sed -n 's/.*release \\([0-9]*\\).*/\\1/p' /etc/redhat-release) && /output/hiveutil.rhel${major_version} install-manager --work-dir /output --log-level debug %s %s", cd.Namespace, provisionName)},
```

Comment:
TODO: it would be nice if we could determine the rhel version earlier in this flow, so we could copy only the necessary hiveutil binary above, and always run it as `hiveutil` here. I would think we could do it from the release metadata, but that would mean we need to do it from the updateinstaller job... which may not always run?

Please open a backlog card for this.